pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -16,8 +16,8 @@ func TestBootstrappedConstraints(t *testing.T) {
 	// ordering of expectedConstraintNames is important, we check it against scc.ByPriority
 	expectedConstraintNames := []string{
 		SecurityContextConstraintsAnyUID,
-		SecurityContextConstraintsHostNetwork,
 		SecurityContextConstraintRestricted,
+		SecurityContextConstraintsHostNetwork,
 		SecurityContextConstraintNonRoot,
```

Comment:
I'd expect that `nonroot` is stricter than `hostnetwork`. Hm. Is it because `nonroot` doesn't require to drop any capabilities?

Reply:
`SecurityContextConstraintNonRoot` in https://github.com/openshift/origin/blob/master/pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go#L92 has `securityapi.RunAsUserStrategyMustRunAsNonRoot` while `SecurityContextConstraintsHostNetwork` in https://github.com/openshift/origin/blob/master/pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go#L240 has `securityapi.RunAsUserStrategyMustRunAsRange`.

And in https://github.com/openshift/origin/blob/master/pkg/security/securitycontextconstraints/byrestrictions.go#L72 and https://github.com/openshift/origin/blob/master/pkg/security/securitycontextconstraints/byrestrictions.go#L37, the values are

```
runAsNonRootPoints points = 30000
runAsRangePoints points = 20000
```

That's why the `hostnetwork` with `AsRange` ends up being less strict than `nonroot` with `AsNonRoot`.

Of course, the question is why `runAsRangePoints` < `runAsNonRootPoints` which has been that way since 6db81779c852b5285b6b07ede964ec8436a5dc64.

Reply:
Hmm, that was just a renaming commit. The original logic comes from 89d91534576bf23c5d8ca77b830a17e0a86bd180. @pweil-, would you remember why it had

```
+       case kapi.RunAsUserStrategyMustRunAsNonRoot:
+               points += 3
+       case kapi.RunAsUserStrategyMustRunAsRange:
+               points += 2
```

?

Reply:
non-root is more "relaxed" because it is up to the user.  MustRunAsRange means the admin has made a decision on the user's behalf.  That said, the fact that non-root is not dropping caps is something that raises my eyebrow.  

Anything that allows host access should be considered more dangerous that things that don't.  Separate bug though than what @adelton is addressing here.

Reply:
> That said, the fact that non-root is not dropping caps is something that raises my eyebrow.

I've created https://github.com/openshift/origin/issues/16371 for this.